Gemfile

```diff
@@ -1,3 +1,3 @@
 source 'https://rubygems.org'
-
+gem 'gpgme'
```

Comment:
We don't put requirements in Gemfile, they're in spec file. But I would not want to mandate installing ```gpgme```. I would put it in ```CVS#setup``` as ```require 'gpgme' if @cfg.gpg```, so  that we only load it, if user is actually using it.